src/Shared/Resources/Strings.shared.resx

```diff
@@ -278,10 +278,14 @@
     <value>MSB5028: Solution filter file at "{0}" includes project "{1}" that is not in the solution file at "{2}".</value>
     <comment>{StrBegin="MSB5028: "}UE: The project filename is provided separately to loggers.</comment>
   </data>
+  <data name="DebugPathTooLong" Visibility="Public">
+    <value>MSB5029: The path "{0}" used for debug logs is too long. Set it to a shorter value using the MSBUILDDEBUGPATH environment variable or change your system configuration to allow long paths.</value>
```

Comment:
If we mention changing the system to allow long paths, should we also include a help link?

Reply:
Reasonable, but that's a very google-able not-MSBuild thing, so I'd vote no, personally.